source/operations/concepts/scanner.rst

```diff
@@ -0,0 +1,95 @@
+.. _minio-concepts-scanner:
+
+==============
+Object Scanner
+==============
+
+.. default-domain:: minio
+
+.. contents:: Table of Contents
+   :local:
+   :depth: 1
+
+Overview
+--------
+
+MinIO uses the built-in scanner to check objects for healing and to take any scheduled object actions.
+Such actions may include:
+
+- calculate data usage on drives
+- apply configured :ref:`lifecycle management <minio-lifecycle-management>` or :ref:`object retention <minio-object-retention>` rules
+- :ref:`bucket <minio-bucket-replication>` or :ref:`site <minio-site-replication-overview>` replication
+- check objects for missing or corrupted data or parity shards and perform :ref:`healing <minio-concepts-healing>`
+
+The scanner performs these functions at two levels: cluster and bucket.
+At the cluster level, the scanner splits all buckets into sets and scans one set at a time.
+The scanner starts with new buckets, then randomizes the scanning of other buckets.
+The scanner completes checks on all bucket sets before starting over with a new set of scans.
+
+At the bucket level, the scanner groups items in buckets and scans selected items from that bucket.
+The scanner selects objects for a scan based on a hash of the object name.
+Over a span of 16 scans, MinIO checks every object it knows about.
+
+Scan Length
+-----------
+
+Multiple factors impact the time it takes for a scan to complete.
+
+Some of these factors include:
+- Type of hard drives provided to MinIO
+- Throughput available
+- Number and size of objects
+- Other activity on the MinIO Server
+
+For example, MinIO may pause the scanner to make I/O operations available for read and write requests.
```

Comment:
```suggestion
For example, by default MinIO will pause the scanner to make I/O operations available for read and write requests.
```

Reply:
I have a thing agains `will` as almost always overpromising, but adjusting the phrasing to get it closer to your suggestion.